CHANGELOG.md

```diff
@@ -172,7 +174,8 @@ a long list of states to go through.
 * Support for data in states
 * Change log added
 
-[Unreleased]: https://github.com/korken89/smlang-rs/compare/v0.6.0...master
+[Unreleased]: https://github.com/korken89/smlang-rs/compare/v0.7.0...master
+[v0.6.0]: https://github.com/korken89/smlang-rs/compare/v0.6.0...v0.7.0
```

Comment:
Copy paste error

Reply:
Fixed in 9b97da80106831cf6d35edc7a9e53eeb04ebea7c, thanks for the catch